sponsorship.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: page
+title: Sponsorship
+permalink: /sponsorship/
+---
+
+The Colorado Data Science Team hosts several competitions each semester with prizes for the top three teams. 
+<ul>
+ <li> These prizes are often sponsored by a local business or nonprofit organization, and in exchange, we invite the sponsor to come to a team meeting at the beginning or end of the competition to tell us about their organization and how they use, or would like to use, data science or machine learning techniques to advance their goals.</li>
```

Comment:
For markdown files should stuck to using markdown since thats sort of the point.